### ISSUE TITLE
A11y_AzureDataStudio_Summary_ScreenReader:NVDA does not announces the name of the table.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/azuredatastudio#7657]
"[Check out Accessibility Insights!](https://accessibilityinsights.io) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**#A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#SQL Azure Data Studio;#WCAG1.3.1;#WCAG4.1.2;#DesktopApp;#Win10;#NVDA;#JAWS;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
NVDA Version:2019.2

**Repro Steps:**
1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster”.
5.	Check the check box I accept in the “select the deployment option” screen and click on select button.
6.	Select any block and click on next button and again click on next button.
7.	Enter password and confirm password and click on next button and again click on next button.
8.	summary screen is opened. 
9.    Start the NVDA and and check NVDA announces the table name or not.

**Actual:**
NVDA screen reader does not announces the table name.

**Expected:**
NVDA screen reader should  announces the table name.

**Same issue observed in JAWS screen reader.**

**User Impact:**
Screen reader user does not get name of the regarding the control.

**Attachment for Reference:**
[7657_A11y_AzureDataStudio_Summary_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3715949/7657_A11y_AzureDataStudio_Summary_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-May2019;
